lib/services/navigation_service.dart

```diff
@@ -62,6 +64,78 @@ class NavigationService {
     );
   }
 
+  void showTalawaErrorWidget(
+    String message, {
+    Duration duration = const Duration(seconds: 2),
+  }) {
+    ScaffoldMessenger.of(navigatorKey.currentContext!).showSnackBar(SnackBar(
+      padding: EdgeInsets.zero,
+      content: Row(
+        children: [
+          Container(
+            width: 20,
+            height: 80,
+            decoration: const BoxDecoration(color: Colors.red),
+          ),
+          const SizedBox(
+            width: 10,
+          ),
+          const Icon(
+            Icons.error,
+            color: Colors.red,
+            size: 35,
+          ),
+          const SizedBox(
+            width: 10,
+          ),
+          Expanded(
+            flex: 1,
+            child: SingleChildScrollView(
+              scrollDirection: Axis.horizontal,
+              child: Text(
+                message,
+                style: const TextStyle(color: Colors.white),
+              ),
+            ),
+          )
+        ],
+      ),
+      // backgroundColor: Colors.grey,
+      backgroundColor: const Color.fromRGBO(65, 65, 66, 1),
+    ));
+  }
+
+  void showTalawaErrorDialog(String errorMessage) {
+    showDialog(
```

Comment:
Do the same with this error dialog please.

Reply:
Done the same for it too.